dev/devicelab/bin/tasks/hot_mode_dev_cycle_linux__preview_dart_2_benchmark.dart

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2017 The Chromium Authors. All rights reserved.
```

Comment:
For this and the other new file to take effect, you would also need to add the corresponding entries in the `manifest.yaml` file.

Reply:
Done

Reply:
I also realized I probably don't need linux/win flavors of preview_dart_2 tests. Having copy of just the base one would be sufficient. Not sure what OS it runs on though, got to be different from linux/win otherwise why to have special flavors of the tests for linux/win.